File: infrastructure/src/main/java/com/github/dkurata38/liblio/infrastructure/domain/librarian/LibrarianRecordMapper.java
package com.github.dkurata38.liblio.infrastructure.domain.librarian;

import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Param;

@Mapper
public interface LibrarianRecordMapper {
    LibrarianRecord selectByUsername(@Param("username") String username);
}

File: infrastructure/src/main/kotlin/com/github/dkurata38/liblio/infrastructure/librarian/LibrarianRecordMapper.kt
package com.github.dkurata38.liblio.infrastructure.librarian

import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Param

@Mapper
interface LibrarianRecordMapper {
    fun selectByUsername(@Param("username") username: String?): LibrarianRecord?
}
